presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/query/PinotFilterExpressionConverter.java

```diff
@@ -141,6 +246,27 @@ private PinotExpression handleCast(CallExpression cast, Function<VariableReferen
             if (typeManager.canCoerce(input.getType(), expectedType)) {
                 return input.accept(this, context);
             }
+            // Special handling for Date type casting. Expression like `DATE '2014-01-31'` is not cast to daysSinceEpoch
```

Comment:
Should the above comment read: 

Expression like `DATE '2014-01-31'` is not cast to a constant number (like days since epoch) and thus it needs to be specifically handled here.

I feel that the daysSinceEpoch reference is ambiguous: Because you also refer it to mean a Pinot column name in other places.

Reply:
make sense, changed column to `eventDate`, `eventTimestamp` to be distinguish from the value type